src/spdx/parser/rdf/rdf_parser.py

```diff
@@ -62,6 +65,21 @@ def translate_graph_to_document(graph: Graph) -> Document:
                 logger.extend(err.get_messages())
         parsed_fields[element] = elements
 
+    for element, triple, relationship_type in [
+        ("has_files", (None, SPDX_NAMESPACE.hasFile, None), RelationshipType.CONTAINS),
+        ("describes_package", (None, SPDX_NAMESPACE.describesPackage, None), RelationshipType.DESCRIBES),
+    ]:
+        for parent_node, _, element_node in graph.triples(triple):
+            try:
+                relationship = parse_implicit_relationship(
+                    element_node, graph, parent_node, creation_info.document_namespace, relationship_type
```

Comment:
In this line PyCharm complains about a wrong type: "Expected type 'URIRef', got 'Node' instead". I don't know how to solve this, the return type of `graph.triples(..)` is `Tuple[Node, Node, Node]`,  `parse_implicit_relationship` expects a `URIRef`. Any suggestions how to fix are much appreciated.

Reply:
I think your proposal is the right way to go. We can't expect a "random" triple from the parsed RDF graph to be a `URIRef`, so we have to make sure that we get the correct type. I'm curious, though, why this doesn't pop up in other uses of `graph.triples()`.

Reply:
I added a wrapper method for the `graph.triples(..)` and also for `graph.value(..)` to ensure types at some places. I decided to write a Warning in the logger if the type doesn't match as the code would probably run either way.